docs/docsite/rst/conf.py

```diff
@@ -309,7 +355,11 @@
 # (source start file, target name, title, author, document class
 # [howto/manual]).
 latex_documents = [
-    ('index', 'ansible.tex', 'Ansible Documentation', AUTHOR, 'manual'),
+    (
+        'index', 'ansible.tex', 'Ansible Documentation', AUTHOR, 'manual',
+    ) if tags.has('core_lang') else (
+        'index', 'ansible.tex', 'Ansible 2.2 Documentation', AUTHOR, 'manual',
```

Comment:
The 2.2 should also go, but again, this wasn't introduced by this PR, so let's do it in a follow-up.